ait/gui/static/js/ait/dtype.js

```diff
@@ -206,7 +206,11 @@ class Time32Type extends TimeType
 class Time64Type extends TimeType
 {
     decode (raw) {
-        const parts = String(raw).split('.').map(x => parseInt(x))
+        let parts = [0, 0]
```

Comment:
Is '0' the only case where a decimal would be missing?  If not, should the check be for '.', if found, usual logic, otherwise parts = [ parseInt(raw), 0] ?

Reply:
That's a very good question @nttoole. I'll poke

Reply:
Updated in line with your recommendation @nttoole

If `raw % 1 === 0` it's not a float and we need to handle that edge case. Otherwise we stick with the normal approach.

Good catch